nvflare/fuel/utils/class_utils.py

```diff
@@ -108,18 +98,12 @@ def create_classes_table(self):
                                         not name.startswith("_")
                                         and inspect.isclass(obj)
                                         and obj.__module__ == module_name
+                                        and issubclass(obj, FLComponent)
                                     ):
-                                        # same class name exists in multiple modules
-                                        if name in scan_result_table:
-                                            scan_result = scan_result_table[name]
-                                            if name in class_table:
-                                                class_table.pop(name)
-                                                class_table[f"{scan_result.module_name}.{name}"] = module_name
-                                            class_table[f"{module_name}.{name}"] = module_name
+                                        if name in class_table:
+                                            class_table[name].append(module_name)
```

Comment:
Is it possible that the module_name already exists in the list?

Reply:
The same module_name should not be existed in the list. This is the same package and same file name. Should not happen.